README.md

```diff
@@ -48,7 +48,7 @@ Coin | src | Version/Tree | Status | dPoW
 KMD | [komodo](https://github.com/komodoplatform/komodo) | 0.6.0 | [![dPOW Status](http://116.203.64.110/svg/KMD_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-Mainnet
 BTC | [bitcoin](https://github.com/bitcoin/bitcoin) | 0.16 | [![dPOW Status](http://116.203.64.110/svg/KMD_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-Mainnet
 AXO | [komodo](https://github.com/komodoplatform/komodo) | 0.6.0 | [![dPOW Status](http://116.203.64.110/svg/AXO_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
-AYA | [aryacoin](https://github.com/sillyghost/AYAv2) | [aa6d65b (v2.1.0)](https://github.com/sillyghost/AYAv2/tree/aa6d65b412954fea3063215daecac96cf03fee03) | [![dPOW Status](http://116.203.64.110/svg/AYA_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-3P
+AYA | [aryacoin](https://github.com/sillyghost/AYAv2) | [fd94422 (AYAv2.1.0)](https://github.com/sillyghost/AYAv2/tree/aa6d65b412954fea3063215daecac96cf03fee03) | [![dPOW Status](http://116.203.64.110/svg/AYA_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-3P
```

Comment:
`fd94422` would be
`https://github.com/sillyghost/AYAv2/tree/fd94422aff2886919dc963d85c313df4dfb0d770`
not
`https://github.com/sillyghost/AYAv2/tree/aa6d65b412954fea3063215daecac96cf03fee03`
and has anyone actually checked it or are we just saying `fd94422` because it's the latest with the keys??

Reply:
> fd94422 would be
https://github.com/sillyghost/AYAv2/tree/fd94422aff2886919dc963d85c313df4dfb0d770
not
https://github.com/sillyghost/AYAv2/tree/aa6d65b412954fea3063215daecac96cf03fee03

I had already corrected it.that's the commit above your review.

> and has anyone actually checked it or are we just saying fd94422 because it's the latest with the keys??

the latest commits update the hardfork height and nothing else.